src/utils.js

```diff
@@ -48,7 +48,11 @@ export function mapPathString(nodePath, state) {
     return;
   }
 
-  const modulePath = getRealPath(nodePath.node.value, state);
+  const sourcePath = nodePath.node.value;
+  const currentFile = state.file.opts.filename;
+  const opts = state.opts;
+
+  const modulePath = resolvePath.call(state, sourcePath, currentFile, opts);
```

Comment:
Technically it _can_ be stateless; this is just a crutch so that the original `resolvePath` can reuse the normalized options if they are there. Without `state` as `this` the function will just normalize by itself.

I agree that it may be over-optimized (normalizing seems to be quite cheap). Do you think it's better to get rid of this for clarity?